src/api/libraryApi.ts

```diff
@@ -59,3 +59,14 @@ export function addToLibrary(gameId: string, libraryId: string): Promise<string
         .catch(err => console.log(err));
 }
 
+export function renameLibrary(name: string, libraryId:string): Promise<Library> {
+    return fetch(homeUrl+librariesBaseUrl+`${libraryId}`, {
```

Comment:
do we want spaces between the +? I don't care too much about this compared to the colons but maybe worth it